src/utils.jl

```diff
@@ -18,24 +18,24 @@ function build_component_funcs(symex)
 end
 
 function modelingtoolkitize_expr(ex::Expr,vars,curmod)
-    names = [x.op.name for x in vars]
+    names = [tosymbol(x) for x in vars]
     ex.head === :if && (ex = Expr(:call, ifelse, ex.args...))
     ex.head === :call || throw(ArgumentError("internal representation does not support non-call Expr"))
-    op = ex.args[1] ∈ names ? vars[findfirst(x->ex.args[1] == x.op.name,vars)] : getproperty(curmod,ex.args[1]) # HACK
-    args = Expression[modelingtoolkitize_expr(x,vars,curmod) for x in ex.args[2:end]]
-    return Operation(op, args)
+    op = ex.args[1] ∈ names ? vars[findfirst(x->ex.args[1] == tosymbol(x),vars)] : getproperty(curmod,ex.args[1]) # HACK
+    args = [modelingtoolkitize_expr(x,vars,curmod) for x in ex.args[2:end]]
+    return Term(op, args)
 end
 
-function modelingtoolkitize_expr(ex::Variable,vars,curmod)
-    convert(Operation,ex)
+function modelingtoolkitize_expr(ex::Sym,vars,curmod)
+    ex
 end
 
 function modelingtoolkitize_expr(ex::Symbol,vars,curmod)
-    names = [x.op.name for x in vars]
-    op = ex ∈ names ? vars[findfirst(x->ex == x.op.name,vars)] : getproperty(curmod,ex) # HACK
-    convert(Expression,op)
+    names = tosymbol.(vars)
+    op = ex ∈ names ? vars[findfirst(x->(ex == tosymbol(x)),vars)] : getproperty(curmod,ex) # HACK
```

Comment:
```suggestion
    op = ex ∈ names ? vars[findfirst(x->ex == x, names)] : getproperty(curmod,ex)
```
I think this also works?